driver/CMakeLists.txt

```diff
@@ -53,7 +53,9 @@ add_executable(MIOpenDriver
     dm_tensorop.cpp
     main.cpp
     registry_driver_maker.cpp
-    rocrand_wrapper.cpp)
+    rocrand_wrapper.cpp
+    
+    dm_pad_constant.cpp)
```

Comment:
Please change the location of dm_pad_constant.cpp above dm_pool.cpp.

Reply:
I've sorted the CMakeLists in alphabetical order as specified